.ado/pipelines/templates/jobs-workload-deploy.yaml

```diff
@@ -160,6 +162,7 @@ jobs: # using jobs so they each run in parallel
 
     - task: AzureCLI@2
       displayName: 'Install workload BackgroundProcessor on AKS clusters'
```

Comment:
not sure if you want this set in all tasks for consistency - we've already inline retry logic for the deployment of catalogservice and backgroundprocessor (due to the issues with cert-manager)

Reply:
hm valid point. would would you prefer?

Reply:
it probably makes sense to set it consistenly across all tasks as you've suggested - inline is still required when a delay between two retries is required

Reply:
ok lets keep it as is then